utils/maintenence.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
```

Comment:
On that note, this seems to do *all* maintenance tasks at once.  This seems like a bad model.  It would be better if it let you choose which one to do or to select "all"  to do them all. I would say that it is pretty rare during normal development to want to do them all.